.gitignore

```diff
@@ -17,4 +17,9 @@ out/
 # Concourse
 aviator_pipeline.yml
 
-*.jar
\ No newline at end of file
+*.jar
+
+terraform/modules/pre-auth-lambda/*.zip
```

Comment:
This doesn't exist anymore (Dragos just merged a change)

Reply:
Removed